modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -599,6 +614,112 @@ def test_reorder_labels_dtypes():
     df_equals(result.dtypes, result.to_pandas().dtypes)
 
 
+@pytest.mark.parametrize(
+    "left_partitioning, right_partitioning, ref_with_cache_available, ref_with_no_cache",
```

Comment:
```suggestion
    "left_partitioning, right_partitioning, res_with_cache_available, res_with_no_cache",
```

Reply:
'ref' means reference value, the naming was on purpose